Dockerfile

```diff
@@ -22,20 +22,20 @@ FROM node:6-alpine
 LABEL xo-server=5.18.3 \
          xo-web=5.18.2
 
-ENV USER=node \
-    USER_HOME=/home/node \
+ENV USER=root \
+    USER_HOME=/root \
     XOA_PLAN=5 \
     DEBUG=xo:main
 
-WORKDIR /home/node
+WORKDIR /root
 
-RUN apk add --no-cache tini su-exec bash util-linux
+RUN apk add --no-cache tini su-exec bash util-linux nfs-utils openrc
```

Comment:
I see the add of `nfs-utils` (which I understand we need the nfs tools to use NFS) and `openrc`  ? why the  `openrc` package ?

Reply:
openrc is required to run the nfs service. Without it there are mounting errors the prevent the mount from happening.